cli/terminal.ts

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+
+const installCommand = (commandPath: string, commandName: string): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    if (typeof commandPath !== 'string' || typeof commandName !== 'string') {
+      reject(new TypeError('Expected a string'));
+    }
+
+    if (process.platform !== 'darwin') {
```

Comment:
can we make this work on other platforms (i.e. windows)? or would that require a different approach?

Reply:
The current approach will work only on macOS. It will require a different approach to support Windows. First, I am planning to support macOS and make everything working well. I am actively working on this feature right now. I may end today by supporting Windows too.